.github/workflows/ci.yml

```diff
@@ -76,12 +76,13 @@ jobs:
           go-version: 1.17.x
       - name: make test
         run: make test
-  e2e:
-    name: e2e test
+  kubernetes:
+    name: Kubernetes
```

Comment:
Why change the job name?
I'd prefer the previous name, like:
- CI Workflow / e2e test(vx.y.z) (pull_request)